templates/common/render/render.go

```diff
@@ -117,6 +117,7 @@ func Render(ctx context.Context, p *Params) (outErr error) {
 		outErr = errors.Join(outErr, tempRemover.maybeRemoveAll(ctx))
 	}()
 
+	logger.DebugContext(ctx, "render phase: downloading/copying template")
```

Comment:
Same with these - they are already namespaced, so I think you can drop "render phase: ".

Reply:
Done.